Example/AztecUITests/AztecUITests.swift

```diff
@@ -0,0 +1,263 @@
+        richContentTextView.tap()
+        richContentTextView.enterAndSelectText(text: "text")
+
+        app.scrollViews.otherElements.buttons[elementStringIDs.boldButton].tap()
+        app.buttons[elementStringIDs.sourcecodeButton].tap()
+
+        htmlContentTextView = app.textViews[elementStringIDs.htmlTextField]
+        let text: String = htmlContentTextView.value as! String
+        let expected = "<strong>text</strong>"
+        XCTAssertEqual(expected, text)
+    }
+
+    func testSimpleItalicText() {
+        richContentTextView = app.textViews[elementStringIDs.richTextField]
+        richContentTextView.tap()
+        richContentTextView.enterAndSelectText(text: "text")
+
+        app.scrollViews.otherElements.buttons[elementStringIDs.italicButton].tap()
+        app.buttons[elementStringIDs.sourcecodeButton].tap()
+
+        htmlContentTextView = app.textViews[elementStringIDs.htmlTextField]
+        let text: String = htmlContentTextView.value as! String
+        let expected = "<em>text</em>"
+        XCTAssertEqual(expected, text)
+    }
+
+    func testSimpleUnderlineText() {
+        richContentTextView = app.textViews[elementStringIDs.richTextField]
+        richContentTextView.tap()
+        richContentTextView.enterAndSelectText(text: "text")
+
+        app.scrollViews.otherElements.buttons[elementStringIDs.underlineButton].tap()
+        app.buttons[elementStringIDs.sourcecodeButton].tap()
+
+        htmlContentTextView = app.textViews[elementStringIDs.htmlTextField]
+        let text: String = htmlContentTextView.value as! String
+        let expected = "<u>text</u>"
+        XCTAssertEqual(expected, text)
+    }
+
+    func testSimpleStrikethroughText() {
+        richContentTextView = app.textViews[elementStringIDs.richTextField]
+        richContentTextView.tap()
+        richContentTextView.enterAndSelectText(text: "text")
+
+        app.scrollViews.otherElements.buttons[elementStringIDs.strikethroughButton].tap()
+        app.buttons[elementStringIDs.sourcecodeButton].tap()
+
+        htmlContentTextView = app.textViews[elementStringIDs.htmlTextField]
+        let text: String = htmlContentTextView.value as! String
+        let expected = "<del>text</del>"
+        XCTAssertEqual(expected, text)
+    }
+
+    func testSimpleBlockquoteText() {
+        richContentTextView = app.textViews[elementStringIDs.richTextField]
+        richContentTextView.tap()
+        richContentTextView.enterAndSelectText(text: "text")
+
+        app.scrollViews.otherElements.buttons[elementStringIDs.blockquoteButton].tap()
+        app.buttons[elementStringIDs.sourcecodeButton].tap()
+
+        htmlContentTextView = app.textViews[elementStringIDs.htmlTextField]
+        let text: String = htmlContentTextView.value as! String
+        let expected = "<blockquote>text</blockquote>"
+        XCTAssertEqual(expected, text)
+    }
+
+    // Enable this test after unordered lists are fully implemented
+    func testSimpleUnorderedListText() {
```

Comment:
Per our Slack conversation, we'll revisit/expand the tests for lists once lists are fully implemented and changes in visual mode affect the HTML (https://github.com/wordpress-mobile/WordPress-Aztec-iOS/issues/150). I'll leave the list tests as-is for now.